commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -157,10 +203,9 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
     }
 
     // To access the service below, you may need to add endpoint =
-    // https://b2b-commerce-test.herokuapp.com in Setup > Security > Remote site settings.
+    // httpHost in Setup > Security > Remote site settings.
```

Comment:
What is this HttpHost for? shipping or tax calculation?

Reply:
The httpHost is something that need to be added to the Remote site settings in Setup so that the calls can be made.

Reply:
https://help.salesforce.com/s/articleView?id=sf.configuring_remoteproxy.htm&type=5